docs/book/src/topics/custom-images.md

```diff
@@ -318,7 +318,7 @@ That's it! You should now have a CAPZ cluster running Mariner Linux.
 [image-builder]: https://github.com/kubernetes-sigs/image-builder
 [image-builder-azure]: https://github.com/kubernetes-sigs/image-builder/tree/master/images/capi/packer/azure
 [kubeadm-preflight-checks]: https://github.com/kubernetes/kubeadm/blob/master/docs/design/design_v1.10.md#preflight-checks
-[mariner]: https://microsoft.github.io/CBL-Mariner/docs/
+[mariner]: https://microsoft.github.io/azurelinux/docs/
```

Comment:
If you go to this link, it looks very ugly because their CSS is failing to load because it's still referring to the old repo name and thus also 404-ing. I'm trusting that that'll get fixed soon enough though.